pkg/resources/results.go

```diff
@@ -53,8 +61,18 @@ func GetResult(resultSpec v1alpha1.ResultSpec, name, namespace, backend string)
 		},
 	}
 }
-func CreateOrUpdateResult(ctx context.Context, c client.Client, res v1alpha1.Result) (ResultOperation, error) {
+func CreateOrUpdateResult(ctx context.Context, c client.Client, res v1alpha1.Result, k8sgptConfig v1alpha1.K8sGPT) (ResultOperation, error) {
 	var existing v1alpha1.Result
+
```

Comment:
@prometherion do we need to set again the labels, since you moved them to `MapResults` ?

Reply:
Whoopsie, I need to remove this! 😁